src/GovernorCountingFractional.sol

```diff
@@ -219,7 +228,7 @@ abstract contract GovernorCountingFractional is Governor {
         _proposalVotes[proposalId] = _proposalVote;
     }
 
-    uint256 constant internal _VOTEMASK = 0xffffffffffffffffffffffffffffffff; // 128 bits of 0's, 128 bits of 1's
+    uint256 constant internal _HALF_WORD_RIGHT = 0xffffffffffffffffffffffffffffffff; // 128 bits of 0's, 128 bits of 1's
```

Comment:
nit: not actually a huge fan of removing "mask" from the name here, especially since we use it within assembly, it could be hard for someone to immediately recognize what we're doing with it. Whereas if you see a name with "mask" it's pretty easy to guess we're using it to mask off some bits.

Reply:
sgtm

Reply:
Went with that in a61d7b9